.ci/travis/travis.sh

```diff
@@ -208,7 +208,7 @@ all-sevntu-checks)
     | grep -E "Check$" \
     | sort | uniq | sed "s/Check$//" > web.txt
   # temporal ignore list
-  sed -i.backup '/Jsr305Annotations/d' web.txt
+  # sed -i.backup '/Jsr305Annotations/d' web.txt
```

Comment:
Do we need to keep this?

Reply:
I would keep it, to not test suppression approach (kind of the same as we have with pitest) if we trap in such problem again.